assets/js/ajax-filters.js

```diff
@@ -175,15 +183,13 @@ jQuery( document ).ready( function ( $ ) {
 		} );
 	} );
 
-	$( '#search_keywords, #search_location, .job_types :input, #search_categories, .job-manager-filter' ).change( function() {
-		var target   = $( this ).closest( 'div.job_listings' );
-		target.triggerHandler( 'update_results', [ 1, false ] );
-		job_manager_store_state( target, 1 );
-	} )
+	//end of the initial update_results listener
 
-	.on( "keyup", function(e) {
+	// Changed this to just listen for enter button instead of listening for change event which was calling the loop twice.
```

Comment:
I think this will be less reliable. Users don't need to hit enter to get results. They could for example just enter text and tab out of the field. Change is better IMO